flink-cyber/metron-parser-chain/parser-chains-config-service/frontend/parser-chains-client/src/app/chain-page/components/chain-view/chain-view.component.ts

```diff
@@ -67,4 +72,8 @@ export class ChainViewComponent implements OnInit {
   collapseExpandAllParsers() {
     this.chainPageService.collapseExpandAllParsers();
   }
+
+  selectedSourceChanged(newSelectedSource: string) {
```

Comment:
When overriding the ngModelChange method, you must also assign the ngModel variable in it, as it will not be assigned automatically 
```suggestion
  selectedSourceChanged(newSelectedSource: string) {
  this.selectedSource = newSelectedSource
```